src/test/java/com.example.online_shop_api/PasswordServiceTest.java

```diff
@@ -0,0 +1,160 @@
+package com.example.online_shop_api;
+
+import com.example.online_shop_api.Entity.Employee;
+import com.example.online_shop_api.Entity.User;
+import com.example.online_shop_api.Repository.EmployeeRepository;
+import com.example.online_shop_api.Repository.UserRepository;
+import com.example.online_shop_api.Service.PasswordService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class PasswordServiceTest {
```

Comment:
Why do we have 2 classes `PasswordServiceTest`?

Reply:
AdminServiceTest and PasswordServiceTest were written by different people and were in two different folders. Maybe, this duplication occurred when merging the two branches.

Reply:
> Why do we have 2 classes `PasswordServiceTest`?

AdminServiceTest and PasswordServiceTest were written by different people and were in two different folders. Maybe, this duplication occurred when merging the two branches.